services/cache/lru.go

```diff
@@ -217,9 +217,10 @@ func (l *LRU) get(key string, value interface{}) error {
 			*v = &u
 			return err
 		case **model.Session:
-			var s model.Session
+			var s = model.UserSessionPool.Get().(*model.Session)
```

Comment:
We don't need to get the object here. We can get it in the app layer itself and pass the session object in the get call. And then remove this case altogether because now it will get handled by the `value.(msgp.Unmarshaler);` case above.

Reply:
Sure, I will close the PR and reopen when done.